### PR TITLE
[incubator/distribution] added RBAC support

### PR DESCRIPTION
**Pull request by jainishshah17**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by jainishshah17** [comment]
@viglesiasce  Can you please take a look?

**Comment by jainishshah17** [comment]
@eldada

**Comment by jainishshah17** [comment]
@rimusz ping

File: incubator/distribution/README.md [hunk 1]
```diff
@@ -97,6 +101,48 @@ The following table lists the configurable parameters of the distribution chart
...
+To get Helm to create an ingress object with a hostname, add these two lines to your Helm command:
+```
+helm install --name distribution \
+  --set ingress.enabled=true \
+  --set ingress.hosts[0]="distribution.company.com" \
+  --set distribution.service.type=NodePort \
+  incubator/distribution
+```
+
+If your cluster allows automatic creation/retrieval of TLS certificates (e.g. [kube-lego](https://github.com/jetstack/kube-lego)), please refer to the documentation for that mechanism.
```
Comment (rimusz) [inline incubator/distribution/README.md]: Replace `kube-lego` which is depreciated with the `[cert-manager](https://github.com/jetstack/cert-manager)`

Comment (jainishshah17) [reply incubator/distribution/README.md]: @rimusz done

File: stable/distribution/README.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+# Example
+# MongoDB host: custom-mongodb.local
+# MongoDB port: 27017
+# MongoDB user: distribution
+# MongoDB password: password1_X
+
+$ export DISTRIBUTION_MONGODB_CONN_URL='mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@custom-mongodb.local:27017/${MONGODB_DATABSE}'
+$ export DISTRIBUTION_MONGODB_AUDIT_URL='mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@custom-mongodb.local:27017/audit?maxpoolsize=500'
+$ helm install -n distribution --set global.mongoUrl=${DISTRIBUTION_MONGODB_CONN_URL},global.mongoAuditUrl=${DISTRIBUTION_MONGODB_AUDIT_URL} incubator/distribution
```
Comment (mattfarina) [inline stable/distribution/README.md]: This line references incubator but the PR has the chart in stable. Might need to update that.

**Review by mattfarina — CHANGES_REQUESTED** [review]
This chart is currently in incubator but this PR moves it to stable. Can you please follow the process in https://github.com/helm/charts/blob/master/PROCESSES.md#promoting-a-chart-from-incubator-to-stable so we can have a good user experience for consumers.

File: stable/distribution/README.md (hunk 2 above)
Comment (jainishshah17) [reply stable/distribution/README.md]: @mattfarina Should I create new PR to deprecate incubator chart?

Comment (jainishshah17) [reply stable/distribution/README.md]: @mattfarina Here is the new PR for deprecating incubator chart https://github.com/helm/charts/pull/7155

Comment (davidkarlsen) [reply stable/distribution/README.md]: @jainishshah17 ☝️ this still needs a fix

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/distribution/README.md (hunk 2 above)
Comment (jainishshah17) [reply stable/distribution/README.md]: @davidkarlsen Done

**Review by mattfarina — APPROVED** [review]

**Pull request merged**